js_modules/dagit/packages/ui/src/components/StyledButton.tsx

```diff
@@ -4,17 +4,44 @@ import {IconWrapper} from './Icon';
 import {SpinnerWrapper} from './Spinner';
 import {FontFamily} from './styles';
 
+export type ButtonJoinedSide = 'left' | 'right' | 'both';
+
 interface StyledButtonProps {
   $fillColor: string;
   $strokeColor: string;
   $textColor: string;
+  $joinedSide?: ButtonJoinedSide;
 }
 
 export const StyledButton = styled.button<StyledButtonProps>`
   align-items: center;
   background-color: ${({$fillColor}) => $fillColor || 'transparent'};
   border: none;
   border-radius: 8px;
+
+  ${({$joinedSide}) =>
+    $joinedSide
+      ? `
```

Comment:
I think you can skip this outer ternary, which would result in this being an empty string if both inner ternaries are false.